web/lib/schema/index.ts

```diff
@@ -190,3 +190,31 @@ export const appStoreImageSchema = yup
     /^[a-zA-Z]{2}\/[a-zA-Z0-9_-]+\.[a-zA-Z0-9]+$/,
     "Invalid image format",
   );
+
+/** this should be allowed, later on replaced by actual username */
+const NOTIFICATION_USERNAME_SPECIAL_STRING = "${username}";
```

Comment:
```suggestion
const NOTIFICATION_USERNAME_SPECIAL_STRING = "{{username}}";
```

optional: use double brackets, like popular template engines do (https://handlebarsjs.com/)